<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Planta</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>48</y>
     <width>709</width>
     <height>439</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" >
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QDockWidget" name="uiToolDock" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="windowTitle" >
         <string/>
        </property>
        <widget class="QWidget" name="dockWidgetContents" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>20</y>
           <width>70</width>
           <height>399</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QGraphicsView" name="uiView" />
      </item>
      <item>
       <widget class="QFrame" name="frame" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape" >
         <enum>QFrame::VLine</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="uiServer" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="uiPosition" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>(0, 0)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>709</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenImage" />
    <addaction name="separator" />
    <addaction name="actionLogin" />
    <addaction name="actionNewTemplate" />
    <addaction name="actionOpenTemplate" />
    <addaction name="actionSaveTemplate" />
    <addaction name="actionSaveTemplateAs" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <addaction name="actionToggleImageBoxes" />
    <addaction name="actionToggleTemplateBoxes" />
    <addaction name="actionToggleFeatureBoxes" />
    <addaction name="actionToggleBinarized" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionDelete" />
   </widget>
   <widget class="QMenu" name="menu_Actions" >
    <property name="title" >
     <string>&amp;Actions</string>
    </property>
    <addaction name="actionFindMatchingTemplateByOffset" />
    <addaction name="actionFindMatchingTemplateByText" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menu_Actions" />
   <addaction name="menuView" />
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>22</y>
     <width>709</width>
     <height>26</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionToggleImageBoxes" />
   <addaction name="actionToggleTemplateBoxes" />
   <addaction name="actionToggleFeatureBoxes" />
   <addaction name="actionToggleBinarized" />
   <addaction name="separator" />
   <addaction name="actionUnzoom" />
   <addaction name="actionZoom" />
  </widget>
  <action name="actionOpenImage" >
   <property name="text" >
    <string>&amp;Open Image</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionToggleImageBoxes" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>&amp;Show Image Boxes</string>
   </property>
  </action>
  <action name="actionLogin" >
   <property name="text" >
    <string>&amp;Login</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveTemplate" >
   <property name="text" >
    <string>&amp;Save Template</string>
   </property>
  </action>
  <action name="actionOpenTemplate" >
   <property name="text" >
    <string>Open &amp;Template</string>
   </property>
  </action>
  <action name="actionNewTemplate" >
   <property name="text" >
    <string>New Template</string>
   </property>
  </action>
  <action name="actionToggleTemplateBoxes" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Show Template Boxes</string>
   </property>
  </action>
  <action name="actionToggleBinarized" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Show Binarized</string>
   </property>
  </action>
  <action name="actionSaveTemplateAs" >
   <property name="text" >
    <string>Save Template &amp;As...</string>
   </property>
  </action>
  <action name="actionDelete" >
   <property name="text" >
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut" >
    <string>Del</string>
   </property>
  </action>
  <action name="actionZoom" >
   <property name="text" >
    <string>&amp;Zoom</string>
   </property>
  </action>
  <action name="actionUnzoom" >
   <property name="text" >
    <string>Unzoom</string>
   </property>
  </action>
  <action name="actionToggleFeatureBoxes" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Show Feature Boxes</string>
   </property>
  </action>
  <action name="actionFindMatchingTemplateByOffset" >
   <property name="text" >
    <string>Find Matching Template by &amp;Offset</string>
   </property>
  </action>
  <action name="actionFindMatchingTemplateByText" >
   <property name="text" >
    <string>Find Matching Template by &amp;Text</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
